recipes/adaflow/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# build script for conda package
+
+set -e
+set -x
+
+mkdir build && cd build
+cmake \
+      ${CMAKE_ARGS} \
+      -DPKG_CONFIG_EXECUTABLE=$CONDA_PREFIX/bin/pkg-config \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX ..
+make -j${nproc} && \
+make install
```

Comment:
Is there a make check/test or ctest we can run there?

Reply:
To validate installation itself, `gst-inspect-1.0 flow_gst_plugin` wil be enough, as GST will acutally attempt to load the library and report.

Integration tests are written in python, and it requires extra [pyhon wheel](https://pypi.org/project/adaflow-python/) to run.